external/js/cothority/src/calypso/calypso-rpc.ts

```diff
@@ -43,7 +43,7 @@ export class OnChainSecretRPC {
         await ctx.updateCountersAndSign(this.bc, [signers]);
         await this.bc.sendTransactionAndWait(ctx);
         // Ask for the full proof which is easier to verify.
-        const p = await this.bc.getProof(ctx.instructions[0].deriveId());
+        const p = await this.bc.getProofFromLatest(ctx.instructions[0].deriveId());
```

Comment:
The proof is sent to create the LTS. In that situation, it's dangerous to send a short proof as you don't know the state of the server that will receive the message.